### ISSUE TITLE
Common Purpose names should be maintained by WCAG

### ISSUE THREAD

**Issue by chaals** [w3c/wcag#316]
The new success criterion 1.3.4 makes a lot of sense.

(Although "Common Purpose" in english generally means a shared goal, rather than a widely-recognised one. It might be useful to change the name of the Success Criterion to reflect that).

Rather than baking in the list of terms by reference to HTML 5.2, it would make more sense if they were managed as an appendix to WCAG itself, or a parallel Rec-rack document.

That list from autocomplete makes a useful starting point, but it seems likely that there will be a number of changes specific to the accessibility use which might not be so easily adopted for "autofill" - and almost certainly only by extensions rather than the bigger American browsers taking them on natively, so needing to go through an HTML update cycle seems like an unnecessary barrier, introducing an external timeline dependency in the pathway to updates.

Having the information as a part of WCAG allows for a simpler update path for what is basically a vocabulary. An alternative is to make a separate Recommendation - again to allow for simpler updates to suit the needs of WCAG.

**Comment by awkawk** [comment]
@chaals - Can I ask that you open this as an issue in the WCAG21 repo?

**Comment by chaals** [comment]
Sure. Sorry for the mistake... :(
